sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -43,7 +43,7 @@ class StateOperatorProgress private[sql](
     val numRowsTotal: Long,
```

Comment:
Could you also fix `newNumLateInputs` in this file?

Reply:
Sigh seems like I did find with "match case" and missed it. Will find and fix. Thanks!

Reply:
Looks like "match case" missed lots of spots. I fixed all the spots.